docs/input-kafka.asciidoc

```diff
@@ -315,6 +316,17 @@ consumers join or leave the group. The value must be set lower than
 `session.timeout.ms`, but typically should be set no higher than 1/3 of that value.
 It can be adjusted even lower to control the expected time for normal rebalances.
 
+[id="plugins-{type}s-{plugin}-isolation_level"]
+===== `isolation_level` 
+
+  * Value type is <<string,string>>
+  * Default value is `"read_uncommitted"`
+
+Controls how to read messages written transactionally. If set to `read_committed`, `consumer.poll()` will only return
```

Comment:
users might have no idea what `consumer.poll()` means
we should re-phrase and just state that:  
> If set to `read_committed`, polling messages will only return ...

Reply:
Thanks for the suggestion; applied and will push shortly! Had taken this verbatim from the Kafka docs, but agree this kinder wording would help users here.